string robot_name
---
src2_object_detection/DetectedBoxes boxes
